sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -210,15 +210,12 @@ EXPLAIN FORMATTED
 struct<plan:string>
 -- !query output
 == Physical Plan ==
-AdaptiveSparkPlan (9)
-+- HashAggregate (8)
-   +- Exchange (7)
-      +- HashAggregate (6)
-         +- Union (5)
```

Comment:
This is a explain test and I think we do want to check the explain result with Union. How about we change the predicate so that we can't optimize out the Union?